content/sensu-go/5.17/installation/install-sensu.md

```diff
@@ -165,7 +202,7 @@ The web UI provides a unified view of your monitoring events and user-friendly t
 After starting the Sensu backend, open the web UI by visiting http://localhost:3000.
 You may need to replace `localhost` with the hostname or IP address where the Sensu backend is running.
 
-To log in to the web UI, enter your Sensu user credentials (the user ID and password you provided with `sensu-backend init`, or `admin` and `P@ssw0rd!` if you're using Docker).
+To log in to the web UI, enter your Sensu user credentials (the user ID and password you provided with `sensu-backend init`).
```

Comment:
Rather than mention `sensu-backend init`, which runs behind the scenes, it may be better to say something like:

(the user ID and password provided with the `SENSU_BACKEND_CLUSTER_ADMIN_USERNAME` and `SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD` environment variables).